src/exampleTypes/TagsQuery/ActionsMenu.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import { observer } from 'mobx-react'
+import TagsJoinPicker from '../TagsJoinPicker'
+import { withTheme } from '../../utils/theme'
+import { Flex } from '../../greyVest'
+import { copyTags } from './utils'
+
+let ActionsMenu = ({ node, tree, open, theme: { Button, Checkbox } }) => (
+  <Flex
+    style={{ minWidth: 240, padding: 10 }}
+    className="tags-query-actions-menu"
+    column
+    justifyContent="stretch"
+    alignItems="stretch"
+  >
+    {!!_.get('tags.length', node) && (
+      <>
+        <Button
+          onClick={() => {
+            copyTags(node)
+            F.off(open)()
+          }}
+        >
+          Copy Keywords
```

Comment:
I think we also agreed to call everything "tags" instead of keywords (because it could actually be keywords or phrases)